modin/core/dataframe/algebra/binary.py

```diff
@@ -270,6 +270,8 @@ def try_compute_new_dtypes(
 class Binary(Operator):
     """Builder class for Binary operator."""
 
+    _binary_ops = set()
```

Comment:
Let's add a comment on what this is needed for.

Reply:
Done.